virtualbox/machine.go

```diff
@@ -690,13 +690,13 @@ func (m *Machine) Modify() error {
 
 // AddNATPF adds a NAT port forarding rule to the n-th NIC with the given name.
 func (m *Machine) AddNATPF(n int, name string, rule driver.PFRule) error {
-	return vbm("controlvm", m.Name, fmt.Sprintf("natpf%d", n),
+	return vbm("modifyvm", m.Name, fmt.Sprintf("--natpf%d", n),
```

Comment:
I'm definitely also surprised by this change -- does it not work as-is after removing the `--synthcpu` flag?  I'd prefer to have this PR be as conservative as possible (especially so we can merge a fix before 1.7.1 which is supposed to happen sometime today), and perhaps a new PR to bring us better parity with `docker-machine` afterwards (but I'd even more prefer that we encourage users to just use `docker-machine` directly instead).